dhis-2/dhis-support/dhis-support-commons/src/main/java/org/hisp/dhis/commons/config/jackson/ParseDateStdDeserializer.java

```diff
@@ -44,6 +44,14 @@
     @Override
     public Date deserialize( JsonParser parser, DeserializationContext context ) throws IOException
     {
-        return DateUtils.parseDate( parser.getValueAsString() );
+        try
+        {
+            return DateUtils.parseDate( parser.getValueAsString() );
```

Comment:
It seems `DateUtils.parseDate` can also return null if parsing went wrong, do we need to handle that?